### PR TITLE
Reader: Fix site results header lines

### PR DESCRIPTION
**Pull request by jancavan**
This fixes: https://github.com/Automattic/wp-calypso/issues/15264

**Before:**
screenshot 2017-06-19 11 52 23

**After:**
screenshot 2017-06-19 11 52 33

### CODE REVIEW

**Comment by fraying** [comment]
Perfecto! Thanks, @jancavan 🚢

File: client/reader/search-stream/style.scss [hunk 1]
```diff
@@ -319,14 +319,14 @@ $reader-post-card-breakpoint-small: "( max-width: 550px )";
 
 .search-stream__header .section-nav {
 	background: inherit;
-	border-bottom: 2px solid lighten( $gray, 30% );
+	border-bottom: 1px solid lighten( $gray, 20% );
 	box-shadow: none;
-	height: 54px;
+	height: 53px;
```
Comment (samouri) [inline client/reader/search-stream/style.scss]: unrelated fix?

**Comment by samouri** [comment]
Nice 🎉

File: client/reader/search-stream/style.scss (hunk 1 above)
Comment (jancavan) [inline client/reader/search-stream/style.scss]: No, had to limit height of `.section-nav`, otherwise, the underline for active tab won't sit perfectly on the gray line:
screenshot 2017-06-19 14 51 44
